src/pages/Encounters/tabs/EncounterNotesTab.tsx

```diff
@@ -153,11 +153,11 @@ const MessageItem = ({ message }: { message: Message }) => {
         <TooltipProvider>
           <Tooltip>
             <TooltipTrigger asChild>
-              <div className="flex-shrink-0">
+              <div className="flex-shrink-0 w-8 h-8 bg-secondary-300 rounded-full">
                 <Avatar
                   name={message.created_by.username}
                   imageUrl={message.created_by.profile_picture_url}
-                  className="w-8 h-8 rounded-full object-cover"
+                  className="object-cover"
```

Comment:
let's keep the size and rounded-full defined in avatar's classname itself like how it's done everywhere else. also why apply bg color? Doesn't avatar already apply a randomized bg-color automatically?